src/components/heroCanvas.js

```diff
@@ -58,8 +58,6 @@ const drawRotatingBrandmark = (canvas, heroImageTemplate) => {
   const drawClip = timestamp => {
     if (!didWindowResize(window)) {
       fabric.util.requestAnimFrame(drawClip)
-    } else {
-      canvas.clear()
```

Comment:
Don't bother clearing the old canvas since (1) it only clears the background image and not the rotating brandmark and (2) it won't animate any further anyway and should be garbage-collected soon after.